tasktiger/redis_scripts.py

```diff
@@ -263,6 +263,63 @@
     return result
 """
 
+# KEYS = { }
+# ARGV = { n_commands
+#          [, cmd_1_n_args, cmd_1_name, [cmd_1_arg_1 , ..., cmd_1_arg_n]
+#            [, ...
+#              [, cmd_n_n_args, cmd_n_name, [cmd_n_arg_1 ..., cmd_1_arg_n]]]]}
+#
+# Example: ARGV = { 2, 1, "GET", "key", 0, "INFO" }
+EXECUTE_PIPELINE = """
```

Comment:
Since this is a non-trivial Lua script what about storing it in a separate file `execute_pipeline.lua` and pulling loading it in once. Would have some syntax highlighting benefits.